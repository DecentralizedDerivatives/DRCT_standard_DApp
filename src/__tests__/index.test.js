import ReactDOM from 'react-dom';

ReactDOM.render = jest.fn();

describe('application initialization', () => {
  it('renders application to DOM', () => {
    require('../index');
    expect(ReactDOM.render.mock.calls).toMatchSnapshot();
  });
});
