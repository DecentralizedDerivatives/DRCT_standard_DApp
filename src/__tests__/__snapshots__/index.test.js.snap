// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`application initialization renders application to DOM 1`] = `
Array [
  Array [
    <Root>
      <Connect(AppRouter) />
    </Root>,
    null,
  ],
]
`;
